docs/Salesforce-streamingsource.md

```diff
@@ -19,14 +19,18 @@ The plugin can either create a PushTopic for the user, or use an existing one.
 Configuration
 -------------
 
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
+
+**Connection:** Name of the connection to use.
+
 **Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
 
 **Username:** Salesforce username.
 
 **Password:** Salesforce password.
 
-**Security Token:** Salesforce security Token. If the password does not contain the security token the plugin 
-will append the token before authenticating with salesforce.
+**Security Token:** Salesforce security Token. If the password does not contain the security token, the plugin 
```

Comment:
change Salesforce security Token to Salesforce security token

Reply:
done